content/courses/native-onchain-development/program-state-management.md

```diff
@@ -554,17 +554,19 @@ We're ready to build and deploy our program!
 You can test your program by submitting a transaction with the right instruction
 data. For that, feel free to use
 [this script](https://github.com/Unboxed-Software/solana-movie-client) or
```

Comment:
This URL needs to be updated to https://github.com/solana-developers/movie-review-program-client.

It's already been partially updated but it does need a format.

Reply:
okay, I'd work on that

Reply:
Just created a pull request which fixes the mentioned issues. 

https://github.com/solana-developers/movie-review-program-client/pull/1

Reply:
> This URL needs to be updated to https://github.com/solana-developers/movie-review-program-client and that repo needs to be updated too. It's already been partially updated but it does need a bunch of other small updates (format, import specific items instead of '*', remove IIFE wrappers, use esrun, etc). - basically what you did already for the frontend repo.

url updated ✅